astropy/table/tests/test_mixin.py

```diff
@@ -11,8 +11,8 @@
 from astropy import coordinates, time
 from astropy import units as u
 from astropy.coordinates import EarthLocation, SkyCoord
+from astropy.coordinates.representation.tests.test_representation import representation_equal
 from astropy.coordinates.tests.helper import skycoord_equal
-from astropy.coordinates.tests.test_representation import representation_equal
```

Comment:
Isn't this technically breaking API?

Reply:
I didn't think they were.

Reply:
When different subpackages import from it, it is kinda semi-public. There is a chance downstream package test might be using it too but I am not sure.

Reply:
I think this goes away if the tests stay in place; maybe another reason to think about splitting up `representations` and moving the tests separately.

Reply:
As in 2 separate PRs?

Reply:
Yes, sorry that I wasn't clear -- I think the splitting of the module is easy as long as the tests still run. The moving of the tests I'm not sure is best, but if we do it, might make sense to reorganize a bit.

Reply:
✅